Gemfile

```diff
@@ -18,9 +20,15 @@ end
 group :development do
   gem 'rbs'
   gem 'rubocop'
+  gem 'racc'
```

Comment:
What's this for?

Reply:
required for rbs to work here and CI to pass.  Maybe not required anymore (may have been a temporarily missing dependncy in rbs?)

Reply:
I get this error without it:
```
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
/Users/alexwoo/.gem/ruby/3.3.0/gems/parser-3.1.2.0/lib/parser.rb:12:in `require': cannot load such file -- racc/parser (LoadError)
```

Reply:
Edit: racc is a bundled gem in 3.3, so needs to be included going forward: https://stdgems.org